modules/meteroid/crates/diesel-models/src/invoices.rs

```diff
@@ -31,6 +31,7 @@ pub struct InvoiceRow {
     pub last_issue_error: Option<String>,
     pub data_updated_at: Option<NaiveDateTime>,
     pub invoice_date: NaiveDate,
+    pub total: i64,
```

Comment:
this should be changed to Option here and in schema.rs as NotNull was dropped in the migration?

Reply:
hm I guess it's an issue on the migration side and we should make it mandatory, as it's considered mandatory everywhere on the application level